db/migrate/20220916043854_delete_and_recreate_mhv_opt_in_flags.rb

```diff
@@ -0,0 +1,26 @@
+class DeleteAndRecreateMHVOptInFlags < ActiveRecord::Migration[6.1]
+  def up
+    drop_mhv_opt_in_flags_table
+    create_mhv_opt_in_flags_table
+  end
+
+  def down
+    drop_mhv_opt_in_flags_table
+    create_mhv_opt_in_flags_table
```

Comment:
For `down`, you should use the definition from the previous migration, just to keep it consistent:

>     create_table :mhv_opt_in_flags do |t|
>      t.string :user_account_uuid, null: false
>       t.string :opt_in_flag, null: false
>
>       t.index [ "user_account_uuid" ],  unique: false
>       t.index [ "opt_in_flag" ],  unique: false
>       t.timestamps
>     end